.github/workflows/build.yaml

```diff
@@ -0,0 +1,39 @@
+name: build
+
+on: [push, pull_request]
+jobs:
+  build:
+    name: build
+    strategy:
+      matrix:
+        go-version: [1.16, 1.17, 1.18]
+        os: [ubuntu-latest]
```

Comment:
ubuntu-latest is currently Ubuntu 20.04, but will likely change to 22.04 by end of year.

This should be okay, just wanted to make a note of it.